seatunnel-dist/src/main/assembly/assembly-bin-ci.xml

```diff
@@ -118,6 +118,47 @@
             </excludes>
             <outputDirectory>/lib</outputDirectory>
         </fileSet>
+        <!-- seatunnel engine -->
+        <fileSet>
+            <directory>../seatunnel-engine/seatunnel-engine-client/target</directory>
+            <includes>
+                <include>seatunnel-engine-*.jar</include>
+            </includes>
+            <excludes>
+                <exclude>%regex[.*((javadoc)|(sources))\.jar]</exclude>
+            </excludes>
+            <outputDirectory>/lib</outputDirectory>
+        </fileSet>
+        <fileSet>
+            <directory>../seatunnel-engine/seatunnel-engine-common/target</directory>
```

Comment:
The `seatunnel-engine-common` and `seatunnel-engine-core` both contains in client and server jar. So we don't should single put it into lib directory.

Reply:
Yes, you are right. I will fix it.